pycaching/geocaching.py

```diff
@@ -513,3 +512,18 @@ def my_dnfs(self, limit=float('inf')):
         :param limit: The maximum number of results to return (default: infinity).
         """
         return self.my_logs(LogType.didnt_find_it, limit)
+
+
+class SortOrder(enum.Enum):
+    """Enum of possible cache sort orderings returned in Groundspeak API."""
+    # NOTE: extracted from https://www.geocaching.com/play/map/public/main.2b28b0dc1c9c10aaba66.js
+    containersize = "containersize"
```

Comment:
I am not sure, but shouldn't we name the variables using underscores to better comply with the general code style?

Reply:
Should I also use all upper-case for enum names?

https://docs.python.org/3/library/enum.html#creating-an-enum

Reply:
As far as I have seen, all our enums use lower-case names, so we should probably stick with this for now.

Reply:
I would prefer using underscores as @FriedrichFroebel says.

Having all enums upper-case would be nice, but it's a major interface change and it's not worth it.

Reply:
fixed